modules/byo-vpc-single-az.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * osd_cluster_create/managing-byo-vpc-single-az.adoc
+
+[id="byo-vpc-single-az_{context}"]
+= Configuring subnets for a VPC with a single availability zone
+
+For VPCs with a single availability zone, you must configure a `public` subnet and `private` subnet, and then, create the appropriate routing rule and NAT gateway for this availability zone.
```

Comment:
See above.

Reply:
here, appears to be more general use of 'public' and 'private'

Reply:
You don't need a comma after "then"